clients/unshaded/src/main/java/tachyon/client/RemoteBlockInStream.java

```diff
@@ -163,6 +173,13 @@ public void close() throws IOException {
       mCheckpointInputStream.close();
     }
     mClosed = true;
+    if (mBytesReadRemote > 0) {
+      mTachyonFS.incBytesReadRemote(mBytesReadRemote);
+      mTachyonFS.incBlocksReadRemote(1);
+    }
+    if (mBytesReadUfs > 0) {
+      mTachyonFS.incBytesReadUfs(mBytesReadUfs);
```

Comment:
This is not very necessary since we now use user heartbeat to pass the data to worker instead of direct RPC calls after every read operations. I can call `TachyonFS.incBytesReadRemote` directly for any remote read operation. But it seems we still need keep the counter `mBytesReadRemote` so that we know if we need call `incBlocksReadRemote(1)`.